src/main/java/com/direwolf20/buildinggadgets/common/items/gadgets/GadgetGeneric.java

```diff
@@ -170,7 +169,7 @@ public static void toggleConnectedArea(EntityPlayer player, ItemStack stack) {
         player.sendStatusMessage(new TextComponentString(TextFormatting.AQUA + new TextComponentTranslation("message.gadget.connected" + suffix).getUnformattedComponentText() + ": " + getConnectedArea(stack)), true);
     }
 
-    protected static String formatName(String name) {
+    public static String formatName(String name) {
```

Comment:
why?

Reply:
Will use a different strategy that makes translation a method in the interface.